roles/variables/defaults/main.yml

```diff
@@ -2115,12 +2154,18 @@ kafka_connect_replicator_group_id: replicator
 ### Variable to define the location of the Embedded Rest Proxy for configuring RBAC.
 kafka_connect_replicator_erp_host: localhost:8090
 
-### Set this variable to the user name of the admin user for the Embedded Rest Proxy, to configure RBAC.
+### Set this variable to the user name of the Ldap admin user for the Embedded Rest Proxy, to configure RBAC.
```

Comment:
where does this get used?

Reply:
It gets used for giving the role bindings to replicator in ldap mode